src/lbpDashboard/lbpDashboard.html

```diff
@@ -5,6 +5,7 @@
   <require from="./sparkChart/spark-chart"></require>
   <require from="./timeRemaining/timeRemaining"></require>
   <div class="page animated-page au-animate lbpDashboardContainer">
+    <div show.to-view="loading" class="loading"><i class="fas fa-circle-notch fa-spin"></i></div>
```

Comment:
looks to me like this will affect the entire dashboard.  It should only affect the chart, and only when the chart data is being fetched on page load. See in attached:

```
 if (!lbpmgr.priceHistory) {
        this.lbpMgr.ensurePriceHistory();
      }
```